build_scripts/build_utils.py

```diff
@@ -11,6 +11,8 @@
 OUT_DIR = os.path.join(PYTYPE_SRC_ROOT, "out")
 SRC_PYI_DIR = os.path.join(PYTYPE_SRC_ROOT, "pytype", "pyi")
 OUT_PYI_DIR = os.path.join(OUT_DIR, "pytype", "pyi")
+CMAKE_LOG = os.path.join(OUT_DIR, "cmake.log")
```

Comment:
I've been wondering, is there a way to see the logs from a Travis run? I see /home/travis/... paths, but there's no obvious way to access them.

Reply:
As discussed offline, we should setup uploading build artifacts to GCS so that we can download and see the logs.